node/index.ts

```diff
@@ -1,17 +1,17 @@
 import 'bluebird-global'
 
-import { hrToMillis, MetricsAccumulator } from '@vtex/api'
+import './globals'
+
+import { hrToMillis, Service } from '@vtex/api'
 import { map } from 'ramda'
 
-import { dataSources, initialize } from './dataSources'
+import { clients } from './clients'
 import { canonical } from './middlewares/canonical'
 import { customSitemap } from './middlewares/customSitemap'
 import { robots } from './middlewares/robots'
 import { sitemap } from './middlewares/sitemap'
 import { userSitemap } from './middlewares/userSitemap'
-import { Context, Middleware } from './utils/helpers'
-
-(global as any).metrics = new MetricsAccumulator()
+import { Middleware } from './utils/helpers'
 
 Promise.config({
```

Comment:
We are stopping using bluebird-global but directly importing it explicitly instead